### PR TITLE
[Closes #24]

### LINKED ISSUE #24: Add pull to refresh in latest issue fragment.

### PR DESCRIPTION
**Pull request by furkanaskin**

### CODE REVIEW

File: latest-issue/src/main/java/net/androidweekly/latestissue/LatestIssueFragment.kt [hunk 1]
```diff
@@ -59,9 +61,13 @@ class LatestIssueFragment : BaseFragment() {
...
-        cardViewLocalIssuesMessage = view?.findViewById(R.id.card_view_fragment_latest_issues_local_issues_message)
-        constraintLayoutParent = view?.findViewById(R.id.constraint_layout_fragment_latest_issues_parent)
-        buttonLocalIssuesRetry = view?.findViewById(R.id.button_fragment_latest_issues_local_issues_retry)
+        cardViewLocalIssuesMessage =
+            view?.findViewById(R.id.card_view_fragment_latest_issues_local_issues_message)
+        constraintLayoutParent =
+            view?.findViewById(R.id.constraint_layout_fragment_latest_issues_parent)
+        buttonLocalIssuesRetry =
+            view?.findViewById(R.id.button_fragment_latest_issues_local_issues_retry)
+        swipeRefreshLayout = view?.findViewById(R.id.swipeRefreshLayout)
```
Comment (mnayef95) [inline latest-issue/src/main/java/net/androidweekly/latestissue/LatestIssueFragment.kt]: Please use 120 for hard wrap

File: latest-issue/src/main/java/net/androidweekly/latestissue/LatestIssueFragment.kt [hunk 2]
```diff
@@ -81,6 +87,11 @@ class LatestIssueFragment : BaseFragment() {
             getIssues()
             adapter?.notifyDataSetChanged()
         }
+
+        swipeRefreshLayout?.setOnRefreshListener {
+            getIssues()
+            swipeRefreshLayout?.isRefreshing = false
```
Comment (mnayef95) [inline latest-issue/src/main/java/net/androidweekly/latestissue/LatestIssueFragment.kt]: You should set `isRefreshing` to false when the http response returns

File: latest-issue/src/main/res/layout/fragment_latest_issue.xml [hunk 3]
```diff
@@ -1,77 +1,82 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.swiperefreshlayout.widget.SwipeRefreshLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (mnayef95) [inline latest-issue/src/main/res/layout/fragment_latest_issue.xml]: SwipeRefreshLayout should be wrapped to RecyclerView, not the whole screen

**Review by mnayef95 — CHANGES_REQUESTED** [review]

**Comment by furkanaskin** [comment]
I also added a boolean for handling progressbar and swipeRefresh visibility. Because when i tried to use resource.show for set swipeRefreshLayout.isRefreshing, progressBar and swipeRefresh visible is for user in the same time. @mnayef95

**Review by mnayef95 — APPROVED** [review]

**Pull request merged**